qa/automation/CareerPortal.Tests.Scenarios/5. Hyperlinks/5.1. Hyperlinks on the page.feature

```diff
@@ -0,0 +1,51 @@
+Feature: 5.1 hyperlinks on the page
+Background:
+	Given User opened Career Portal
+	And user hovers over a hyperlink
```

Comment:
the same idea for 'hover over a hyperlink' in background and 'moved to hyperlink in test steps, seems to be excess